src/material/button/_button-theme.scss

```diff
@@ -229,8 +228,21 @@
 }
 
 @mixin typography($theme) {
-  @include mdc-helpers.using-mdc-typography($theme) {
-    @include mdc-button.without-ripple($query: mdc-helpers.$mdc-typography-styles-query);
+  .mat-mdc-button {
+    $typography-tokens: tokens-mdc-text-button.get-typography-tokens($theme);
+    @include mdc-button-text-theme.theme($typography-tokens);
```

Comment:
@mmalerba can confirm, but I think we're supposed to emit these under `sass-utils.current-selector-or-root` rather than the buttons themselves.

Reply:
I assume that's correct so I made the change

I did it this way initially because I wrongly assumed emitting the tokens under the same selector would cause collisions but it's actually ok because they have unique prefixes